backend/tests/conftest.py

```diff
@@ -30,7 +30,7 @@ async def language_fra(book_dict):
 @pytest.fixture(scope="function")
 def book_dict():
     return {
-        "uuid": uuid.uuid4().hex,
+        "id": str(uuid.uuid4()),
```

Comment:
The model says that we're storing the UUID format is string.

Reply:
Yes, actually we use `id` terminology in library.xml so it makes more sense like this. 👍